Casks/testcontainers-desktop.rb

```diff
@@ -24,15 +24,18 @@
                    args: ["#{appdir}/Testcontainers Desktop.app"]
   end
 
-  uninstall delete: [
-              "~/Library/Caches/AtomicJar",
```

Comment:
We probably can delete the whole folder on uninstall, no?

Reply:
Uninstall happens when upgrading as well, so that's exactly the issue we want to avoid